lib/shoryuken/fetcher.rb

```diff
@@ -32,7 +32,7 @@ def fetch(queue, available_processors)
           limit = batch ? FETCH_LIMIT : available_processors
 
           if (sqs_msgs = Array(receive_messages(queue, limit))).any?
-            logger.info { "Found #{sqs_msgs.size} messages for '#{queue}'" }
+            logger.debug { "Found #{sqs_msgs.size} messages for '#{queue}'" }
```

Comment:
@Sof31t I'm in favor of your change. I'm just wondering about people that are used to those logs, they might think something wrong is going on if they stop seeing it after a bump.